docker-compose.dev.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.2"
+
+services:
+  db:
+    image: postgres:12.3
+    container_name: nimble
```

Comment:
Should we add `_db` suffix 📝

Reply:
Fixed that in https://github.com/nimblehq/elixir-templates/pull/9/commits/3be8913e8e8017ff7d6ebf7e52b22266adef3dd0 @rosle